Graphs/Table/index.js

```diff
@@ -218,12 +266,16 @@ class Table extends AbstractGraph {
          * On data change, resetting the paging and filtered data to 1 and false respectively.
          */
         this.resetFilters((currentPage || 1), this.selectedRows);
-
+        const filterColumns = this.getColumnByContext(columns);
         const removedColumns = [];
 
         // remove un-selected columns
         columns.forEach((d, index) => {
-            if (selectedColumns && selectedColumns.length) {
+            if (filterColumns.length) {
```

Comment:
it seems that this is the same code as the static method getRemovedColumns, please re-use